### ISSUE TITLE
Restructure and clean up inputs and gqueries

### ISSUE THREAD

**Issue by mabijkerk** [quintel/etsource#2858]
Some of the inputs and gqueries on ETSource are outdated. It would be useful to clean them up, to avoid clients using inputs and gqueries that are no longer valid.

For example:

- These inputs https://github.com/quintel/etengine/pull/1317
- This gquery https://github.com/quintel/etsource/blob/master/gqueries/general/energy_production/electricity_produced_from_chps_minus_central_coal_chp.gql

**Comment by mabijkerk** [comment]
Additionally, given the increasing importance of the API, our clients are using the ETSource folders themselves to find specific queries. This increases the need for:

- A clear set of generic queries, especially for final demand, primary demand etc.
- A clear folder structure for queries
- Potentially documentation for queries

Notifying @ChaelKruip

**Comment by Charlottevm** [comment]
Nice idea @mabijkerk, would actually also be nice for our own use!

**Comment by mabijkerk** [comment]
Added to this list is to restructure and link the queries that are used by the Transition Paths tool. Currently, they use separate queries.

This causes difficulties, for example in the agriculture sector where the edge group method to query final demand does include geothermal but the node group method does not. The queries in the ETM therefore use the edge group method, but for some reason, completely new queries have been written for the Transition Paths tool. It would be much better if the queries in the Transition Paths tool refer to the same queries that are used in the ETM.

**Comment by kaskranenburgQ** [comment]
Some input statements that are still used in the datasets but not by the user:
area:
- areable_land.ad
-  buildings_roof_surface_available_for_pv.ad
- co2_emission_1990_aviation_bunkers
- co2_emission_1990_marine_bunkers
- co2_emission_1990
- coast_line
- number_of_buildings_both --> Not even used in any datasets
- number_of_buildings_present --> Not even used in any datasets
- number_of_inhabitants_present--> Not even used in any datasets
- offshore_suitable_for_wind.ad
- residences_roof_surface_available_for_pv.ad
-
Costs
- investment_costs_batteries.ad
- investment_costs_electric_boilers.ad

-

**Comment by kaskranenburgQ** [comment]
I created a script to see if queries where referenced. 
I have checked for all queries if they where referenced to in the etmodel, etsource and mechanical turk repositories. 
Around 1300 queries where not referenced in these repositories, I will not post them here, since this will blow up this post. 
@mabijkerk Are there any other repositories I should check with this script, and where should I post the unreferenced queries?
